tests/test_startup.py

```diff
@@ -0,0 +1,11 @@
+import os
+import platform
+import pytest
+
+from ilastik_scripts.ilastik_startup import fix_ssl
+
+
+@pytest.mark.skipif(platform.system() == "Windows", reason="Windows does not need this fix")
+def test_startup_ssl_paths():
+    fix_ssl()
+    assert os.path.isfile(os.environ["SSL_CERT_FILE"])
```

Comment:
Maybe you could check first that `ssl.get_default_verify_paths()` isn't finding the certs and that it _does_ find it after calling `fix_ssl()`

Reply:
It still won't find it after setting SSL_CERT_FILE; the default_verify_paths remain unchanged. I should 1) add a check that SSL_CERT_FILE isn't already set in `ssl_fix`, and 2) for the test I've added the alternative permitted outcomes (so hopefully they should pass CI now)